src/index.d.ts

```diff
@@ -94,11 +113,7 @@ declare namespace connect {
     /*
     * Whether to display the login view.
     */
-    loginPopup?: boolean;
-    /*
-    * Whether to auto-close the CCP popup window.
-    */
-    loginPopupAutoClose?: boolean;
```

Comment:
I think we should've had this in `loginPopup` to begin with, but as this is already in the code, I think we should keep it so it will be backward compatible.

Reply:
Agreed ^^

Reply:
Are you open to accepting either parameter, making it a non-breaking change, but allowing you to document only the preferred implementation, as I suggested [here](https://github.com/amazon-connect/amazon-connect-streams/pull/192#discussion_r371596176)? Then you can easily remove the old way on your next breaking change and most users will see no impact. I can easily add code to accept either and mark it with a comment to remove at the next breaking change. Just tell me how you guys like to tag things like that.

Reply:
Yeah, accepting either parameter seems to me to be the best option. @haochi-aws any opinion here?